docs/docs/getting-started/using-files.md

```diff
@@ -1,46 +1,36 @@
 # Using files
 
-!!! warning "Lug is not a security boundary"
+## Using files locally with Docker images
 
-    We strongly recommend only running trusted code in trusted containers with Lug. It is possible to break out of 
-    Lug's virtualization layer.
+When running a function with Lug locally, the Python function's file access doesn't change. You can skip this section!
 
-## Using files locally
+If you're using a sidecar Docker container, Lug mounts your system's file system at `/lug` inside the container.
 
-When running a function with Lug locally, the Python function's file access does not change. The container – or, in 
-other words, calls to `subprocess.run`, `subprocess.Popen`, and `os.system` – is limited to files in Lug's `mount` 
-argument.
-
-`mount` defaults to the current working directory, but you can pass a new mount point to grant greater access.
+By default, Lug mounts the current working directory to `/lug`. You can change this with the `mount` parameter.
 
 !!! tip "Lug's `mount` is a Docker mount point"
 
     Lug's `mount` is a Docker mount point. This means that Docker must have permission to access the mount point. Some 
     environments, like Docker Desktop on macOS, require explicitly adding permissions for non-user files – even if 
     you're running as `root`.
 
-This grants full root directory access to Lug, but then uses a relative path to access a file one directory above the 
+This snippet grants mounts the root directory, but then uses a relative path to access a file one directory above the 
 current working directory:
```

Comment:
"but then uses a relative path to access a file one directory above the 
current working directory:" not what the example is doing.

Reply:
Fixed